package.json

```diff
@@ -15,5 +15,12 @@
   "bugs": {
     "url": "https://github.com/erizocosmico/npm-audit-lookout-test/issues"
   },
-  "homepage": "https://github.com/erizocosmico/npm-audit-lookout-test#readme"
+  "homepage": "https://github.com/erizocosmico/npm-audit-lookout-test#readme",
+  "dependencies": {
+    "tough-cookie": "<=2.2.0",
+    "tough-cookie-filestore": "0.0.1"
```

Comment:
A vulnerability has been detected on your `package.json`.

* **Vulnerable dependency:** `tough-cookie`
* **Dependency path:** `tough-cookie-filestore>tough-cookie`
* **Dev dependency:** No
* **Severity:** moderate
* **Installed version:** `0.12.1`
* **Current constraint:** `0.12.1`
* **Vulnerable versions:** `>=0.9.7 <=2.2.2`
* **Patched versions:** `>=2.3.0`
* **Recommendation:** Update to version 2.3.0 or later.

Affected versions of `tough-cookie` may be vulnerable to regular expression denial of service when long strings of semicolons exist in the `Set-Cookie` header.

[More info about this vulnerability](https://npmjs.com/advisories/130)